pmg/templates/committee_list.html

```diff
@@ -3,232 +3,289 @@
-                    <label>
-                    {% endif %}
-                    <a href="{{ url_for('committee_detail', committee_id=committee.id) }}" class="name">{{ committee.name }}</a>{% if committee.last_active_year %} <span class="text-muted">({{ committee.last_active_year }})</span>{% endif %}
-                    {% if current_user.is_authenticated() %}
-                    </label>
-                    {% endif %}
-                </form>
-                {% if committee.premium %}
-                  <span class="premium exclude"><i class="fa fa-key"></i></span>
-                {% endif %}
-              </li>
-            {% endfor %}
-            <div class="note not-monitored">This is a select list of committees that PMG is monitoring</div>
-          </ul>
-        </div>
-        {% endfor %}
-      </div>
-    </div>
+	<div class="tab-pane prov cte-items" id="prov">
+	    <div class="row prov-row">
+		{% for key, ctype in wc_committees.iteritems() %}
+		    <div class="col-xs-12 col-md-6 single-prov">
+			<h3>{{ ctype.name }} Parliament</h3>
+			<ul>
+			    {% for committee in ctype.committees %}
+				{% if committee.active %}
+				    <li class="committee item {% if not committee.active %}in{% endif %}active" data-id="{{ committee.id }}" {% if committee.followed %} data-following="true" {% endif %}>
+					{% if committee.followed %}
+					    <form class="cte-follow-committee" action="{{ url_for('user_unfollow_committee', committee_id=committee.id) }}" method="POST">
+					{% else %}
+						<form class="cte-follow-committee" action="{{ url_for('user_follow_committee', committee_id=committee.id) }}" method="POST">
+					{% endif %}
+					<input type="hidden" name="csrf_token" value="{{ csrf_token() }}">
+					<input type="hidden" name="next" value="{{ request.url }}">
+					{% if current_user.is_authenticated() %}
+					    {% if committee.followed %}
+						<input type="checkbox" checked />
+					    {% else %}
+						<input type="checkbox" />
+					    {% endif %}
+					    <label>
+					{% endif %}
+					<a href="{{ url_for('committee_detail', committee_id=committee.id) }}" class="name">{{ committee.name }}</a>{% if committee.last_active_year %} <span class="text-muted">({{ committee.last_active_year }})</span>{% endif %}
+					{% if current_user.is_authenticated() %}
+					    </label>
+					{% endif %}
+						</form>
+						{% if committee.premium %}
+						    <span class="premium exclude"><i class="fa fa-key"></i></span>
+						{% endif %}
+				    </li>
+				{% endif %}
+			    {% endfor %}
+			    <div class="note not-monitored">This is a select list of committees that PMG is monitoring</div>
+			</ul>
+		    </div>
+		{% endfor %}
+	    </div>
+	    <hr>
+	    <div class="row prov-row">
+		{% for key, ctype in wc_committees.iteritems() %}
+		    <div class="col-xs-12 col-md-6 single-prov">
+			<ul>
+			    {% for committee in ctype.committees %}
+				{% if not committee.active %}
+				    <li>
```

Comment:
is this also a repeat of the above? not sure if an include or template tag is more appropriate but some reuse would make the length and indentation/tag closing easier